### PR TITLE
Knex prod install

### PR DESCRIPTION
**Pull request by sreynen**
Some changes for running PostgreSQL on prod.

### CODE REVIEW

**Review by mathemagica — APPROVED** [review]
👍

File: spoke-pm2.config.js.template [hunk 1]
```diff
@@ -9,9 +9,12 @@ const env_production = {
   ASSETS_MAP_FILE:'assets.json',
   CAMPAIGN_ID:'campaign-id-hash',
   DB_HOST:'127.0.0.1',
-  DB_PORT:28015,
+  DB_PORT:5432,
   DB_NAME:'spokedev',
+  DB_USER:'spoke',
+  DB_PASSWORD:'spoke',
   DB_KEY:'',
+  DB_TYPE:'',
```
Comment (schuyler1d) [inline spoke-pm2.config.js.template:17]: make this `DB_TYPE: 'pg',`

**Review by schuyler1d — APPROVED** [review]
:+1:

File: spoke-pm2.config.js.template (hunk 1 above)
Comment (sreynen) [reply spoke-pm2.config.js.template:17]: Do we want to default to PostgreSQL for dev? I was thinking this should be blank by default for SQLite dev.

**Review by shakalee14 — APPROVED** [review]

**Pull request merged**